cmd/restic/cmd_stats.go

```diff
@@ -154,22 +129,17 @@ func runStats(gopts GlobalOptions, args []string) error {
 		return nil
 	}
 
-	// inform the user what was scanned and how it was scanned
-	snapshotsScanned := snapshotIDString
-	if snapshotsScanned == "latest" {
-		snapshotsScanned = "the latest snapshot"
-	} else if snapshotsScanned == "" {
-		snapshotsScanned = "all snapshots"
-	}
-	Printf("Stats for %s in %s mode:\n", snapshotsScanned, countMode)
+	Printf("Stats in %s mode:\n", countMode)
+
+	Printf("Snapshots processed:   %d\n", stats.snapshotsCount)
 
 	if stats.TotalBlobCount > 0 {
-		Printf("  Total Blob Count:   %d\n", stats.TotalBlobCount)
+		Printf("   Total Blob Count:   %d\n", stats.TotalBlobCount)
```

Comment:
Is there a reason to change these statistic output commands?

Reply:
It is only to align the output with the new line containing the number of snapshots that were considered during taking the stats.

Reply:
Ah, I didn't notice from looking at the code alone. Looks fine.